test/kubemark/gce/util.sh

```diff
@@ -46,7 +46,7 @@ function create-kubemark-master {
     KUBE_GCE_INSTANCE_PREFIX="${KUBE_GCE_INSTANCE_PREFIX:-e2e-test-${USER}}-kubemark"
     SCALEOUT_PROXY_NAME="${KUBE_GCE_INSTANCE_PREFIX}-proxy"
     if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]]; then
-      KUBE_GCE_INSTANCE_PREFIX="${KUBE_GCE_INSTANCE_PREFIX}-rp"
+      KUBE_GCE_INSTANCE_PREFIX="${KUBE_GCE_INSTANCE_PREFIX}-rp-${RESOURCE_PARTITION_SEQUENCE}"
```

Comment:
this setting failed to get expected value
if KUBERNETES_TENANT_PARTITION == true, KUBE_GCE_INSTANCE_PREFIX always equals to "${KUBE_GCE_INSTANCE_PREFIX}-tp-${TENANT_PARTITION_SEQUENCE}"

Reply:
it is expected, right ? just as you stated, it does that in line 112. note that when the create-kubemark-master is called, it can ONLY be either RP or TP cluster.

Reply:
Added comment and a invalid combination check to avoid some future confusions on this RP TP settings.